nix-tools-iohk-overlay.nix

```diff
@@ -0,0 +1,45 @@
+commonLib:
+let
+  ghc84 = hackage: {
+    hsc2hs = hackage.hsc2hs."0.68.4".revisions.default;
+    # stackage beautifully omitts the Win32 pkg
+    Win32 = hackage.Win32."2.6.2.0".revisions.default;
+  };
+  ghc86 = hackage: {
+    hsc2hs = hackage.hsc2hs."0.68.4".revisions.default;
+    # stackage beautifully omitts the Win32 pkg
+    Win32 = hackage.Win32."2.6.2.0".revisions.default;
+  };
+in {
+  ghc842 = ghc84;
+  ghc843 = ghc84;
```

Comment:
This was pretty confusing for a bit until I realised that `ghc84` was an overlay.